packages/clarity-visualize/src/enrich.ts

```diff
@@ -43,21 +44,21 @@ export class EnrichHelper {
             node.position = this.position(d.id, d.tag, node, children, children.map(c => this.nodes[c]));
 
             /* For backward compatibility, continue populating current selector and hash like before in addition to beta selector and hash */
-            let input: Layout.SelectorInput = { tag: d.tag, prefix: parent ? [parent.stable, parent.beta] : null, position: node.position, attributes };
+            let input: Layout.SelectorInput = { id: d.id, tag: d.tag, prefix: parent ? [parent.alpha, parent.beta] : null, position: node.position, attributes };
 
             // Get stable selector
             // We intentionally use "null" value for empty selectors to keep parity with v0.6.25 and before.
-            let selector = helper.selector(input);
-            d.selector = selector.length > 0 ? selector : null;
-            d.hash = selector.length > 0 ? helper.hash(d.selector) : null;
+            let selectorAlpha = helper.selector.get(input, Layout.Selector.Alpha);
```

Comment:
Minor: Wuldn't it be a bit cleaner to return both the selectosr and hashes from this function? Too much code dup calling this function and using its values.

Reply:
It's a good idea. I will tackle this in a separate change because it may have impact on external libraries depending on this function.